### DISCUSSION TITLE
How to cite GitPython?

### DISCUSSION THREAD

**Discussion by marie-becker** [gitpython-developers/GitPython#1321] [Ideas]
Hello, 
I need to cite GitPython for my thesis and wondered if you have a prefered citation. I think for the future it would be really nice if you included a citation in bibtex form either in your readme or added a [cff-file](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-citation-files) to your root directory.

Kind regards

**Comment by Byron** [comment]
Thanks for bringing it up, it's the first time.
Personally I don't need citation, mention or attribution of any kind, but also don't mind it.

I'd welcome a PR that adds what would help you in this case, we can figure out the details there then.
